### DISCUSSION TITLE
svs.json discussion

### DISCUSSION THREAD

**Discussion by stakira** [stakira/OpenUtau#433] [Ideas]
Discussion thread for [svs.json: singing voice synthesis backend API](https://github.com/stakira/OpenUtau/wiki/%5BPROPOSAL%5D-svs.json:-singing-voice-synthesis-backend-API)

**Comment by rokujyushi** [comment]
I’d like to know how to handle singer information in order to integrate various SVS systems.
About a year ago, I was considering a project to set up PoinoSing as a server and connect it to other systems.
However, with the emergence of unique SVS systems like klattsch, I realized we should develop a full-fledged singing-voice-synthesis-backend-API.  

[klattsch](https://github.com/tgies/klattsch)
[PoinoSing](https://github.com/rokujyushi/PoinoSing)
